src/components/SingleCard.js

```diff
@@ -98,6 +98,9 @@ const SingleCard = (props) => {
 									<Button
 										variant="outline-info"
 										onClick={() => setOpen(!openIssues)}
+										style={{
+											'-webkit-text-stroke': '0.4px black'
```

Comment:
are we sure about using webkit? Can it cause issues in any of the browsers?

Reply:
I was going by this:
![Screenshot 2024-05-22 015649](https://github.com/version0chiro/Find-Me-Issues/assets/61333980/bd4e5708-f360-4d36-a8f5-c66b93c182b0)

This is on the containing div of the Card. Using `WebkitTextStroke`, shorthand for `-webkit-text-stroke`. Changing that to being black on the button using:
`style={{'-webkit-text-stroke': '0.4px black'}}`

It seems to work in all browsers I have tested. 

Should I be looking elsewhere?